<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Equation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="equation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The equation to be evaluated</string>
       </property>
       <property name="documentTitle">
        <string/>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="plainText">
        <string/>
       </property>
       <property name="tabStopWidth">
        <number>15</number>
       </property>
       <property name="placeholderText">
        <string>Enter Equation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Variables</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="variables"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="varInfo">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Variable Info</string>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="output">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="answerTab">
        <attribute name="title">
         <string>Solution</string>
        </attribute>
        <widget class="QTextBrowser" name="answer">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>451</width>
           <height>231</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Solved Equation</string>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="codeTab">
        <attribute name="title">
         <string>Code</string>
        </attribute>
        <widget class="QTextBrowser" name="python">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>451</width>
           <height>231</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>Equation expressed as Python code</string>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="errorTab">
        <attribute name="title">
         <string>Errors</string>
        </attribute>
        <widget class="QTextBrowser" name="error">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>451</width>
           <height>231</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Problems with the given equation</string>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="load"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="throwError"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="save">
   <property name="text">
    <string>Save Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>Save Program As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="load">
   <property name="text">
    <string>Equation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="throwError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Throw Errors</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
